src/test/resources/features/Account.feature

```diff
@@ -14,17 +14,19 @@ Feature: Manage Account
     When I go to the Dashboard Page
     Then I should see the new account in the Project Creation Popup
 
-#  @deleteAccount @logOut
-#  Scenario: Configure the account name of an account created
-#    When I go to the Account Page
-#      And I create a new account "Test Account2"
-#    Then I should see the new account in Account Plans Page
-#    When I go to the Settings tab inside Account page
-#      And I update the account Name with "Other Name"
-#    Then I should see the new name in account settings page
-#    When I go to the Account Page
-#    Then I should see the new account in list of Accounts page
-##
+  @deleteAccount @logOut
+  Scenario: Configure the account name of an account created
+    When I go to the Account Page
+      And I create a new account "Test Account2"
+    Then I should see the new account in Account Plans Page
+    When I go to the Settings tab inside Account page
+      And I update the account Name with "Other Name"
+    Then I should see the new name in account settings page
+    When I go to the Account Page
+    Then I should see the new account in list of Accounts page
+    When I go to the Dashboard Page
+    Then I should see the new account in the Project Creation Popup
+#
 #  @deleteAccount
```

Comment:
delete commented lines until its funtionality its working.

Reply:
we agreed yesterday to stay there until tomorrow for some to be guided how it has to be written or how they could do their features is a team convention.